Formula/oci-cli.rb

```diff
@@ -43,8 +43,8 @@ class OciCli < Formula
   end
 
   resource "click" do
-    url "https://files.pythonhosted.org/packages/dd/cf/706c1ad49ab26abed0b77a2f867984c1341ed7387b8030a6aa914e2942a0/click-8.0.4.tar.gz"
-    sha256 "8458d7b1287c5fb128c90e23381cf99dcde74beaf6c7ff6384ce84d6fe090adb"
+    url "https://files.pythonhosted.org/packages/27/6f/be940c8b1f1d69daceeb0032fee6c34d7bd70e3e649ccac0951500b4720e/click-7.1.2.tar.gz"
+    sha256 "d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a"
   end
```

Comment:
Why are we downgrading this?

Reply:
that might be due to the fact the oci-cli has constraint for this, let me check.

Reply:
Here you go, https://github.com/oracle/oci-cli/blob/master/requirements.txt#L9

They might added it after last release.

Reply:
This is also in the changelog, https://github.com/oracle/oci-cli/commit/1a31cd373b500c15ae085275b02ec9b88deb9a2c#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR55-R64 :)